index.js

```diff
@@ -1381,7 +1523,7 @@
   //. > Z.concat (Cons ('foo', Cons ('bar', Cons ('baz', Nil))), Cons ('quux', Nil))
   //. Cons ('foo', Cons ('bar', Cons ('baz', Cons ('quux', Nil))))
   //. ```
-  Z.concat = (x, y) => Z.Semigroup.methods.concat (x) (y);
+  Z.concat = (x, y) => Z.Semigroup.methods.concat (x, y);
```

Comment:
```suggestion
  Z.concat = Z.Semigroup.methods.concat;
```

Several other arrow functions could also be dropped.